modules/installation-configuration-parameters.adoc

```diff
@@ -1108,15 +1103,48 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 
-|`platform.gcp.osDisk.diskSizeGB`
+|`platform.gcp.defaultMachinePlatform.zones`
+|The availability zones where the installation program creates machines.
+|A list of valid link:https://cloud.google.com/compute/docs/regions-zones#available[GCP availability zones], such as `us-central1-a`, in a
+link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
+
+|`platform.gcp.defaultMachinePlatform.osDisk.diskSizeGB`
 |The size of the disk in gigabytes (GB).
 |Any size between 16 GB and 65536 GB.
 
-|`platform.gcp.osDisk.diskType`
-|The type of disk.
-|Either the default `pd-ssd` or the `pd-standard` disk type. The control plane nodes must be the `pd-ssd` disk type. The worker nodes can be either type.
+|`platform.gcp.defaultMachinePlatform.osDisk.diskType`
+|The link:https://cloud.google.com/compute/docs/disks#disk-types[GCP disk type].
+|Either the default `pd-ssd` or the `pd-standard` disk type. The control plane nodes must be the `pd-ssd` disk type. Compute nodes can be either type.
+
+|`platform.gcp.defaultMachinePlatform.tags`
+|Optional. Additional network tags to add to the control plane and compute machines.
+|One or more strings, for example "network-tag1".
+
+|`platform.gcp.defaultMachinePlatform.type`
+|The link:https://cloud.google.com/compute/docs/machine-types[GCP machine type] for control plane and compute machines.
+|The GCP machine type, for example "n1-standard-4". 
+
+|`platform.gcp.defaultMachinePlatform.osDisk.encryptionKey.kmsKey.name`
+|The name of the customer managed encryption key to be used for machine disk encryption.
+|The encryption key name.
+
+|`platform.gcp.defaultMachinePlatform.osDisk.encryptionKey.kmsKey.keyRing`
+|The name of the Key Management Service (KMS) key ring to which the KMS key belongs.
+|The KMS key ring name.
+
+|`platform.gcp.defaultMachinePlatform.osDisk.encryptionKey.kmsKey.location`
+|The link:https://cloud.google.com/kms/docs/locations[GCP location] in which the KMS key ring exists.
+|The GCP location.
+
+|`platform.gcp.defaultMachinePlatform.osDisk.encryptionKey.kmsKey.projectID`
+|The ID of the project in which the KMS key ring exists. This value defaults to the value of `platform.gcp.projectID` if not set.
+|The GCP project ID.
+
+|`platform.gcp.defaultMachinePlatform.osDisk.encryptionKey.kmsKey.kmsKeyServiceAccount`
+|The GCP Compute Engine System service account used for the encryption request for both control plane and compute machines. For more information on GCP service accounts, see Google's documentation on link:https://cloud.google.com/compute/docs/access/service-accounts#compute_engine_service_account[service accounts].
```

Comment:
Suggest to replace `The GCP Compute Engine System service account used for the encryption request for both control plane and compute machines. ` with `KMSKeyServiceAccount is the service account being used for the encryption request for the given KMS key. If absent, the Compute Engine default service account is used. `.

Reply:
I would like to leave the mention of "control plane and compute machines" so that the user knows that this global value affects both sets of machines. I have added the sentence about the default service account.